app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -46,12 +46,7 @@ def security_event_tokens
       return {} unless timestamp
 
       events = redis_client.read_events(timestamp: timestamp)
-      sets = {}
-      events.each_pair do |k, v|
-        key_id, jti = k.split(':')
-        sets[jti] = { key_id => v }
-      end
-      sets
+      events.values.join("\r\n")
```

Comment:
Couldn't we just return `events` here, and then modify it where referenced? That way we can cleanly get the count as well. 
```suggestion
      events
```

Reply:
That's arguably more in line with what the method name `security_event_tokens` implies it returns anyway.